src/Acuminator/Acuminator.Utilities/Roslyn/Semantic/PXGraph/DataViewInfo.cs

```diff
@@ -1,10 +1,16 @@
 ﻿using Acuminator.Utilities.Common;
 using Microsoft.CodeAnalysis;
+using System.Collections.Generic;
 
 namespace Acuminator.Utilities.Roslyn.Semantic.PXGraph
 {
-    public readonly struct DataViewInfo
+    public class DataViewInfo
```

Comment:
@metallmonster Can't this class be derived from `GraphOverridableItem`? It looks like it can be a base type for such classes

Reply:
@SENya1990 Not exactly. `DataViewInfo` cannot be derived from `GraphOverridableItem<DataViewInfo>` because in this case constructor of `DataViewInfo` should pass `DataViewInfo` instance (`this`) to the base constructor - `GraphOverridableItem<DataViewInfo>`. And that is not possible.